src/constants/stamps.ts

```diff
@@ -15,7 +20,8 @@ const cachedStamps = new Map<string, StampCollection>();
 
 const getAndCacheStampsFromFile = async (
```

Comment:
As Europe and Australia will share a remote stamp generator, I've renamed this file and just added `stampHost` as a param for europe/australia brand configs to pass in instead.

For Europe it'll be neoPix's repo, for Australia it'll be mine.